engine/src/main/scala/cromwell/engine/workflow/ValidateActor.scala

```diff
@@ -56,67 +55,80 @@ class ValidateActor()
     * 2.) Raw inputs and it's coercion
     * 3.) Workflow Options
     * 4.) RuntimeAttributes w.r.t. a backend
+    *
     * @return The result of the validation as instance of `ValidationResult`
     */
-  private def validateAll(wdlSource: WdlSource, workflowInputs: Option[WdlJson], workflowOptions: Option[WdlJson]): Future[ValidationResult] = {
-    (for {
-      namespaceWithWorkflow <- Future(NamespaceWithWorkflow.load(wdlSource))
-      validatedInputs <- Future(validateInputs(workflowInputs, namespaceWithWorkflow))
-      validatedWorkflowOptions <- Future(validateWorkflowOptions(workflowOptions, CromwellBackend.backend()))
-      validatedRuntimeAttrs <- Future(validateRuntimeAttributes(namespaceWithWorkflow))
-    } yield ValidationSuccess(namespaceWithWorkflow, validatedInputs, validatedWorkflowOptions, validatedRuntimeAttrs)) recover {
-      case reason => ValidationFailure(reason)
+  private def validateAll(wdlSource: WdlSource, workflowInputs: Option[WdlJson], workflowOptions: Option[WdlJson]): Future[ValidationResult] = Future {
```

Comment:
Francisco - if you want to figure out ahead of time what the performance envelope for the ValidationActor is such that the mailbox would get overloaded, the please do so. My only request was that that be understood prior to adding parallelism to the procedure and knowing that in fact it'd fall over under anything approaching realistic load.

OTOH I also said elsewhere in here that if people really want to just add it, go ahead. It's not worth the argument as it would probably need to be put in anyways